### ISSUE TITLE
[Feature request] filter public routes according to current time

### ISSUE THREAD

**Issue by katerynakocherhina** [osmandapp/Osmand#8844]
While using the public transport navigation, among many other options, OsmAnd suggests using the transport which is available during the night time only.

**Comment by vshcherb** [comment]
Yes, it is a known issue, it's kinda duplicate of another issue but will keep it open for now.
